spring-cloud-netflix-core/src/test/resources/application.yml

```diff
@@ -30,6 +30,9 @@ badClients:
   ribbon:
     MaxAutoRetriesNextServer: 10
     ReadTimeout: 200
+username:
+  ribbon:
+    listOfServers: localhost:${local.server.port}
```

Comment:
I got surprised too. But according to my unit tests it does. I'll double check when I'll rework the PR.